api/libproject/serviceaccounts.rb

```diff
@@ -0,0 +1,27 @@
+require_relative "../../libproject/utils/common"
+
+class ServiceAccounts
+
+  CREDENTIALS_BUCKET_NAME = "all-of-us-workbench-test-credentials"
+
+  SA_FILE_NAMES = {
+    :directory_service => {
```

Comment:
I'm assuming that pretty soon we'll have our GCS project configured to use the Gsuite domain of the users... at which point we won't need this anymore. So we probably can just hardcode it to the dev JSON file below.

Reply:
I've just worked through this exercise to determine that the service account accessing the directory API does *not* need to be in the same GSuite domain as the user accounts. With my personal GMail account I created an SA and was able to delegate access to a GSuite domain that I use for testing.

(In the process I also encountered + reported more GSuite management bugs; this stuff is brutal.)

Reply:
This all means we *should* be able to assign user management privileges to our AppEngine SA, regardless of what org/project it lives in. (But to be fair, I haven't tried it this in AppEngine yet...)

Reply:
(I filed b/67066098 internally at Google for this.)

Reply:
So based on Josh's testing, it sounds like we ought to be able to use our application default credentials for creating the users in the test domain... you'd just need to add the service account with the appropriate role in IAM. Is that right?

If so, let's change that here -- there's a lot of code that can be removed.